pkg/system/chtimes_nowindows.go

```diff
@@ -13,3 +15,17 @@ import (
 func setCTime(path string, ctime time.Time) error {
 	return nil
 }
+
+// setAMTimeNoFollow will set access/modification time on a file,
+// without following symbol link.
+func setAMTimeNoFollow(path string, atime time.Time, mtime time.Time) error {
+	uts := []unix.Timespec{
+		unix.NsecToTimespec(atime.UnixNano()),
+		unix.NsecToTimespec(mtime.UnixNano()),
+	}
+	return unix.UtimesNanoAt(unix.AT_FDCWD, path, uts, unix.AT_SYMLINK_NOFOLLOW)
+}
+
+func setCTimeNoFollow(path string, ctime time.Time) error {
+	return nil
+}
```

Comment:
Honestly, I have no idea  how to use window apis, nor I know whether windows support it or not...., the code for windows  meant to be a safe placeholder, `docker save` would resort to original behavior if `nofollow` fails, hence it would not break on windows.    (But I have not tested it......)